hibernate-core/src/main/antlr/sql-script.g

```diff
@@ -57,16 +57,23 @@ options {
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 script
-	: (newLineToSkip)* (statement)* EOF
+	: (newLineToSkip)* (SPACE)* (TAB)* (statement)* EOF
```

Comment:
would the above rule work for `(TAB) * (SPACE)*`, btw?

Reply:
Good catch, I'm going to fix it.

Thanks @NathanQingyangXu

Reply:
It should be fixed now, Thanks @NathanQingyangXu